mailpoet/lib/Automation/Engine/Data/NextStep.php

```diff
@@ -3,16 +3,16 @@
 namespace MailPoet\Automation\Engine\Data;
 
 class NextStep {
-  /** @var string */
+  /** @var string|null */
   protected $id;
 
   public function __construct(
-    string $id
```

Comment:
I just saw this, when I switched to another branch. In support, we sometimes suggest to revert back to an older version. This change means, we can not revert back to an older version. It seems to break the whole automation system. Maybe we could use empty string instead?
![image](https://github.com/mailpoet/mailpoet/assets/6458412/b877582a-62de-4667-af30-f439353d8d90)

Reply:
@websupporter 🤔 An automation with conditions won't work after a downgrade... so I think the bug is rather about where and why do we load the full automation body, outside the editor.

Reply:
@websupporter Ohh, but we can't undo the "bug" in older versions... 😬 I'm hesitant to change the API only to cover this edge case. I mean, it really is just the edge case of 1) using IF/ELSE already, and 2) needing to downgrade after that. If that downgrade fails, it's not the end of the World, they can upgrade back again...